src/pages/index.tsx

```diff
@@ -268,6 +271,49 @@ export default function Home() {
           </div>
         </div>
       </div>
+
+      <div className={styles.Explainer}>
+        <div className={styles.Gradient4 + " " + styles.Right} />
+        <div className={styles.Content}>
+          <h3 className={styles.Pill} id="library">
+            new!
+          </h3>
+          <h1>Javascript Library</h1>
+          <h2>A new Frontier</h2>
+          <p>
+            Made for Developers, 3EM can be used in both client and server side.
+            With our JS library, 3EM can be integrated in a variety of
+            applications while still keeping the integrity of our core
+            principles: Blazingly fast, multi-language, secured & deterministic
+            smart contract executor.
+          </p>
+          <a
+            href="https://github.com/three-em/3em"
```

Comment:
Can we update this link to: https://github.com/three-em/3em/tree/main/js#executecontract

Reply:
done 👍